MetaMask/metamask-extension
# UX: Multichain: Send Flow: Disable account picker for edits

## **Description**

Disables the account picker in the new send flow during an Edit

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/23708?quickstart=1)

## **Related issues**

Fixes: N/A

## **Manual testing steps**

1. `yarn start`
2. Start a send
3. Continue
4. Click Edit to edit the send
5. See the account picker disabled

## **Screenshots/Recordings**

### **Before**

N/A

### **After**

<img width="573" alt="SCR-20240325-mnda" src="https://github.com/MetaMask/metamask-extension/assets/46655/ad185a0c-8fc3-4174-95de-61731f1163e6">


## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
